lua/autorun/server/cfc_destroy_the_gibs.lua

```diff
@@ -1,4 +1,5 @@
-local noGibModels = {
+local maxGibLifetime = 10 -- All serverside gibs will be force-removed after this many seconds.
+local noGibModels = { -- Prevents gibs from ever spawning out of these models.
```

Comment:
We could probably remove the old PropBreak hook stuff now, but iirc those make clientside gibs and thus wouldn't be caught by the new hook. I'm not 100% sure though, I would need to do some additional testing.